Remove partial candle documentation (#3090)
freqtrade/freqtrade: docs/exchanges.md

$ pip3 install web3
```

### Getting latest price / Incomplete candles

Most exchanges return current incomplete candle via their OHLCV/klines API interface.
By default, Freqtrade assumes that incomplete candle is fetched from the exchange and removes the last candle assuming it's the incomplete candle.

> Whether your exchange returns incomplete candles or not can be checked using [the helper script](developer.md#Incomplete-candles) from the Contributor documentation.

Reviewer: ```suggestion
Whether your exchange returns incomplete candles or not - it can be checked using [the helper script](developer.md#Incomplete-candles) from the Contributor documentation.
```

Author: really? to me this sounds strange with the seperating word (when trying to read it out loud...)

Reviewer: > really? to me this sounds strange with the seperating word (when trying to read it out loud...)

no problem, that was only my suggestion...

Due to the danger of repainting, Freqtrade does not allow you to use this incomplete candle.

> However, usually, this requirement is based on the need for the latest price - which can be aquired using the [data provider](strategy-customization.md#possible-options-for-dataprovider) from within the strategy.

Reviewer: ```suggestion
However, if it is based on the need for the latest price for your strategy - then this requirement can be acquired using the [data provider](strategy-customization.md#possible-options-for-dataprovider) from within the strategy.
```
